### PR TITLE
[NCP-3674] added the text recommended

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

**Review by burtonquantil — APPROVED** [review]
The English text looks OK. 
It is interesting that a couple of the translated languages have the word capitalized while the English version is not.

File: zh-Hans.json [hunk 1]
```diff
@@ -299,6 +299,7 @@
 	"certificate.placeholder.share.with": "选择要共享的客户",
 	"certificate.placeholder.sharing.method": "选择共享方式",
 	"certificate.placeholder.state": "输入省、州的名称。",
+	"certificate.public.key.algorithm.recommended.label": "受到推崇的",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:302]: Please change it to "推荐"

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]
The Chinese text needs to be changed

File: zh-Hans.json (hunk 1 above)
Comment (jungho73) [reply zh-Hans.json]: The Chinese text is changed.

**Review by Jianghai-Quantil — APPROVED** [review]
Good to merge now

**Pull request merged**